src/auth/index.js

```diff
@@ -509,7 +509,7 @@ utils.wrapPropertyMethod(
  *   scope: 'openid'  // Optional field.
  * };
  *
- * auth0.oauth.token(data, function (err, userData) {
+ * auth0.oauth.passwordGrant(data, function (err, userData) {
```

Comment:
I think this should be `auth0.passwordGrant`.

Reply:
Why you think it? "passwordGrant" is a method of "OAuthAuthenticator" (auth0.oauth) class, not of "AuthenticationClient" (auth0).

Reply:
https://github.com/auth0/node-auth0/blob/master/src/auth/index.js#L527

This is creating a method on `AuthenticationClient` with the name `passwordGrant` that wraps `oauth.passwordGrant`. Same as we do with all other calls. 

For example see clientCredentialsGrant:
- https://github.com/auth0/node-auth0/blob/master/src/auth/index.js#L465
- https://github.com/auth0/node-auth0/blob/master/src/auth/index.js#L482

Reply:
Ok, thanks for details 👍 
I have edit the row.

Reply:
Thanks you very much !